q2_amr/types/_transformer.py

```diff
@@ -176,7 +176,7 @@ def _11(data: CARDAnnotationDirectoryFormat) -> ProteinsDirectoryFormat:
 
 
 def create_dir_structure(data, seq_type, genes_protein_directory):
-    annotation_dir = str(data)
+    annotation_dir = data
```

Comment:
Can you maybe rather just rename the `data` argument to be the `annotation_dir` and remove this line?